extras/windows/hazelcast-clc-installer.iss

```diff
@@ -0,0 +1,103 @@
+#define MyAppName "Hazelcast CLC"
+#define MyAppVersion "v5.2.0-beta2"
+#define MyAppPublisher "Hazelcast, Inc."
+#define MyAppURL "https://www.hazelcast.com/"

```

Comment:
Maybe we can point users to the GitHub URL here. Wdyt?

Reply:
Let's discuss that after the release.